src/stacks-hierarchy/Root_TabNavigatorStack/TabNav_TicketingStack/Ticketing_TicketTabNavStack/TicketTabNav_PurchaseTabScreen/Components/FareProducts/FareProductTile.tsx

```diff
@@ -71,7 +109,13 @@ export const FareProductTile = ({
             {description}
           </ThemeText>
         </View>
-        <FareProductIllustration style={styles.illustration} config={config} />
+        <FareProductIllustration
+          style={styles.illustration}
+          config={config}
+          fill={transportThemeSecondaryColor.background}
+          width={28}
+          height={28}
```

Comment:
The icon size `large` is 28 in the design system, so I think it's better to get it from `useTheme` here

Reply:
Better indeed